test/core/fork/NFTDescriptor.t.sol

```diff
@@ -17,6 +15,10 @@ contract NFTDescriptor_Fork_Test is Fork_Test {
     IERC20 internal constant DAI = IERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
     address internal constant DAI_HOLDER = 0x66F62574ab04989737228D18C3624f7FC1edAe14;
 
+    ISablierLockup internal lockupDynamic;
+    ISablierLockup internal lockupLinear;
+    ISablierLockup internal lockupTranched;
```

Comment:
Sorry, I didn't understand. Can you please explain or may be jump on a call to explain this?

Reply:
nevermind it, it would be a overkill. we can use `ISablierLockup as the only functions used are `tokenURI` and `nextStreamId` which will most likely be in all protocol versions